recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "tardis-sn" %}
+{% set version = "3.0.a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: True   # [win or py != 36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+  host:
+    - python
+    - pip
+    - setuptools
+    - astropy =3
+    - Cython =0.29
+
+  run:
+    - python
+    - astropy =3
+    - {{ pin_compatible('numpy') }}    
+    - scipy =1.1
+    - pandas =0.24
+    - Cython =0.29
```

Comment:
Is cython needed at runtime?
```suggestion
```

Reply:
I don't think so, but @wkerzendorf will know better.